packages/react-components/react-card/src/stories/CardDefault.stories.tsx

```diff
@@ -15,7 +15,7 @@ export const Default = () => {
   return (
     <Card>
       <CardHeader
-        image={<img src={avatarElviaURL} alt="Face of a person" />}
+        image={{ as: 'img', src: avatarElviaURL, alt: 'Face of a person' }}
```

Comment:
what the reason of this syntax?

Reply:
Avoiding the div wrapper and rendering the `img` element directly